charts/logs/values.yaml

```diff
@@ -65,6 +65,12 @@ fluent-bit:
     grep_match_tag: "nothing"
     grep_exclude: "nomatch ^$"
     inotify_watcher: "true"
+    workers: 2
```

Comment:
This part gives me some doubt - see https://docs.fluentbit.io/manual/v/dev-2.2/pipeline/outputs/stackdriver#occasional-crashing-with-greater-than-1-workers.

Reply:
Oh that's stackdriver. Ignore.